readthedocs/projects/tasks.py

```diff
@@ -1680,6 +1680,20 @@ def remove_dirs(paths):
         shutil.rmtree(path, ignore_errors=True)
 
 
+@app.task()
+def remove_build_storage_paths(paths):
```

Comment:
I'm not sure what we decided here, but I think that build servers don't have the credentials (or the permissions) to make this operation. I think this task may be only executed by webs. In that case, `queue='web'` should be passed in the decorator.

Reply:
It should only be executed by webs typically in response to a project or version being deleted completely. I'll update it.

Reply:
I think you forgot to push this change.